projects/libreoffice/build.sh

```diff
@@ -1,3 +1,20 @@
 #!/bin/bash -eu
 
+# afl++ workaround, this also randomly enables CMPLOG lite
```

Comment:
i thought libreoffice still breaks compile, so if you just want to test, can try in any other project like imagemagick, graphicsmagick (ripe with bugs) [add at start of build.sh].

Reply:
as long as I have not figured out why it fails with afl-clang-fast, the libreoffice change is an interim solution.
the real CMPLOG etc. options will not be in build.sh but in afl_compile and run_fuzzer (or clusterfuzz) - only where afl++ needs a special setup for compilation I would put that in build.sh.